File: app/Models/KeypointExt.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class KeypointExt extends Model
{
    protected $table = 'keypoint_ext';

    protected $primaryKey = 'keypoint_id';

    public $incrementing = false;

    protected $keyType = 'int';

    protected $fillable = [
        'keypoint_id',
        'coordinate',
        'alamat',
        'parent_stationpoints',
    ];
    public $timestamps = false;
}

File: go-backend-pln-monitoring/internal/model/keypoint_ext.go
package model

// KeypointExt represents the keypoint_ext model.
type KeypointExt struct {
	KeypointID        uint   `json:"keypoint_id" gorm:"primaryKey"`
	Coordinate        string `json:"coordinate"`
	Alamat            string `json:"alamat"`
	ParentStationpoints string `json:"parent_stationpoints"`
}
